tests/lib/cmsis_dsp/matrix/src/binary_q15.c

```diff
@@ -206,8 +206,9 @@ DEFINE_TEST_VARIANT5(
 void test_matrix_binary_q15(void)
 {
 	ztest_test_suite(matrix_binary_q15,
-		ztest_unit_test(test_op2_arm_mat_mult_q15),
-		ztest_unit_test(test_op2c_arm_mat_cmplx_mult_q15)
+		ztest_unit_test(test_op2c_arm_mat_cmplx_mult_q15),
+		ztest_unit_test(test_op2_arm_mat_mult_q15)
+
```

Comment:
Nit: please remove the additional blank line

Reply:
Done